redaxo/src/core/lib/minibar/minibar.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * Class for minibar
```

Comment:
Solche Kommentare setze ich in der Regel nicht mehr. Da steckt ja nicht mehr Information drin, als schon in `class rex_minibar`